tools-tested-by-miwg.json

```diff
@@ -116,15 +116,14 @@
       "vendor": "Camunda",
       "tool": "bpmn.io (Camunda Modeler)",
       "maintainer": "nele.uhlemann(at)camunda.com",
-      "version": "9.4.0",
```

Comment:
@barmac The version number is mandatory.

Reply:
It should be 11.5.0. This is a bug in the automation script.